src/machine/board_feather-m0.go

```diff
@@ -0,0 +1,75 @@
+// +build sam,atsamd21,itsybitsy_m0
```

Comment:
This line caused the build to fail. It must be:

```go
// +build sam,atsamd21,feather_m0
```

Reply:
Sorry. I did fix that but my dev env is a bit of a mess and the code I run isn't in the same folder as the git checkout so I need to copy changes back and forth and sometimes lose things. I need to sort that out.

Reply:
I already fixed in `dev` branch, so no further work on this PR required.